modules/opensimplex/config.py

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Please remove leading and trailing newlines (see other `config.py` files).

You could also add definitions to get the documentation included in the module itself:
```
def get_doc_classes():
    return [
        "NoiseTexture",
        "SimplexNoise",
    ]

def get_doc_path():
    return "doc_classes"
```